frontend/src/views/LandingPage.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
+import NavigationBar from '../components/NavBar';
 
 function LandingPage(){
     return(
         <div>
-            {/* Insert your created components here */}
-            <h1>Landing Page Component</h1>
+            <NavigationBar></NavigationBar>
```

Comment:
The navigation bar only needs to go inside of App.js so this line here is unnecessary.